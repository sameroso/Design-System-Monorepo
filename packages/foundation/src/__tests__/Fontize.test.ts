import Fontsize from "../FontSize";
test("snapshot of fontSizes", () => {
  expect(Fontsize).toMatchSnapshot();
});
