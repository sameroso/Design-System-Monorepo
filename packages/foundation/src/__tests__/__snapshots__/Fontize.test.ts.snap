// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot of fontSizes 1`] = `
Object {
  "base": "base",
  "lg": "lg",
  "sm": "sm",
  "xl": "xl",
  "xs": "xs",
}
`;
